CHANGES_NEXT_RELEASE

```diff
@@ -1 +1 @@
-- Set Nodejs 14 as minimum version in packages.json (effectively removing Nodev12 from supported versions)
+- Add: CORS_ENABLED env var (boolean) to enable the cors configuration (of config.js file) in your docker image (#608)

```

Comment:
```suggestion
- Add: CORS_ENABLED env var (boolean) to enable the cors configuration (of config.js file) in your docker image (#608)
- Set Nodejs 14 as minimum version in packages.json (effectively removing Nodev12 from supported versions)
```

This seems to be a wrongly solved merge conflict. You shouldn't delete existing lines in CHANGES_NEXT_RELEASE file, just add yours.